_translator-files/po/es/Custom-Directories.po

```diff
@@ -3,93 +3,94 @@
 # FIRST AUTHOR <EMAIL@ADDRESS>, YEAR.
 #
-#, fuzzy
 msgid ""
 msgstr ""
-"Project-Id-Version: PACKAGE VERSION\n"
-"PO-Revision-Date: YEAR-MO-DA HO:MI+ZONE\n"
-"Last-Translator: FULL NAME <EMAIL@ADDRESS>\n"
-"Language-Team: LANGUAGE <LL@li.org>\n"
-"Language: \n"
+"Project-Id-Version: \n"
+"POT-Creation-Date: \n"
+"PO-Revision-Date: 2022-07-15 17:24+0200\n"
+"Last-Translator: \n"
+"Language-Team: \n"
+"Language: es\n"
 "MIME-Version: 1.0\n"
 "Content-Type: text/plain; charset=UTF-8\n"
 "Content-Transfer-Encoding: 8bit\n"
+"X-Generator: Poedit 3.1.1\n"
 
 #. type: YAML Front Matter: lang
 #: ../wiki/en/Custom-Directories.md:1
 #, no-wrap
 msgid "en"
-msgstr ""
+msgstr "es"
 
 #. type: YAML Front Matter: layout
 #: ../wiki/en/Custom-Directories.md:1
 #, no-wrap
 msgid "wiki"
-msgstr ""
+msgstr "wiki"
 
 #. type: YAML Front Matter: permalink
 #: ../wiki/en/Custom-Directories.md:1
 #, no-wrap
 msgid "/wiki/Custom-Directories"
-msgstr ""
+msgstr "/wiki/Custom-Directories"
 
 #. type: YAML Front Matter: title
 #: ../wiki/en/Custom-Directories.md:1
 #, no-wrap
 msgid "Custom Directories"
-msgstr ""
+msgstr "Directorios Personalizados"
 
 #. type: Plain text
 #: ../wiki/en/Custom-Directories.md:9
 msgid "{% include breadcrumb.html root=\"More\" branch1=\"Server Administration\" branch1-url=\"Running-a-Server\" %}"
-msgstr ""
+msgstr "{% include breadcrumb.html root=\"More\" branch1=\"Administrar un Servidor\" branch1-url=\"Ejecutar un Servidor\" %}"
 
 #. type: Plain text
 #: ../wiki/en/Custom-Directories.md:12
 msgid "# Running a Custom Directory"
-msgstr ""
+msgstr "### Ejecutar un Directorio Personalizado"
 
 #. type: Plain text
 #: ../wiki/en/Custom-Directories.md:14
 msgid "This is a specialised Jamulus Server configuration, as described in [Server Types](Running-a-Server#server-types)."
-msgstr ""
+msgstr "Esta es una configuración especializada del Servidor de Jamulus, descrita en [Tipos de Servidor](Running-a-Server#tipos-de-servidor)."
 
 #. type: Plain text
 #: ../wiki/en/Custom-Directories.md:16
 msgid "To view Servers listed by a Custom Directory, users must enter the address of that Directory in their Client's Settings > Advanced Setup > Custom Directories field. Multiple addresses can be added in this way if needed. Custom Directories will then appear in the Directory drop-down list on their Connect window. Custom Directories otherwise work for Clients in the same way as Public Directories, displaying a list of Servers registered with them."
-msgstr ""
+msgstr "Para ver Servidores listados en un Directorio Personalizado, los usuarios deben introducir en su Cliente la dirección de ese Directorio en Configuración > Configuración Avanzada > campo de Directorios Personalizados. Se pueden añadir múltiples direcciones si se requiere. Por lo demás, los Directorios Personalizados funcionan para los Clientes del mismo modo que los Directorios Públicos, mostrando una lista de Servidores registrados en ellos."
```

Comment:
`To view Servers listed by a Custom Directory, users must enter the address of that Directory in their Client's Settings > Advanced Setup > Custom Directories field.` -> `Para ver Servidores listados en un Directorio Personalizado, los usuarios deben introducir en su Cliente la dirección de ese Directorio en Configuración > Configuración Avanzada > campo de Directorios Personalizados.` ✅ 

`Multiple addresses can be added in this way if needed.` -> `Se pueden añadir múltiples direcciones si se requiere.` ✅ 

`Custom Directories will then appear in the Directory drop-down list on their Connect window.` -> ❓ this appears missing

`Custom Directories otherwise work for Clients in the same way as Public Directories, displaying a list of Servers registered with them.` -> `Por lo demás, los Directorios Personalizados funcionan para los Clientes del mismo modo que los Directorios Públicos, mostrando una lista de Servidores registrados en ellos.` ✅

Reply:
Ok, I can't really check it properly now but I'll look into it.